src/util/modules.py

```diff
@@ -869,6 +869,11 @@ def force_upgrade_of_fresh_module(cr, module, init=True):
     """
     Force the execution of upgrade scripts for a module that is being installed.
 
+    From saas~18.2
+    Odoo Standard Odoo allows to add module names to `registry._config_force_upgrade_scripts`
+    to force the execution of their upgrade scripts. `init=False` is not supported.
```

Comment:
@KangOl
I am not sure if we should support `force_upgrade_of_fresh_module(cr, module, init=False)` in the new strategy.
Could you help me check previously when we set `init=False`, did we really want the data and demo to be loaded in the `update` mode or we just didn't want the `package.init` to be propagated through dependencies.